src/square.js

```diff
@@ -0,0 +1,10 @@
+export function Square({ color, onClick, onHover }) {
```

Comment:
If you do `export default function Square...` I am pretty sure you can import it like 
`import Square from './square'`